handbook/information-security/supplier-policy.md

```diff
@@ -0,0 +1,119 @@
+# Supplier Policy
+
+In providing our services we rely on a number of third party suppliers that have
+access to some of the sensitive information we handle. This policy sets out how
+we vet our suppliers to ensure that we understand and assess the risks involved.
+
+## Objectives
+
+The objectives of our Supplier Policy are to:
+* understand which of our suppliers are handling sensitive information on our
+  behalf
+* understand the risks created by our collaboration
+* ensure the relevant suppliers follow best practices to minimise information
+  security risks
+* work with our suppliers to implement best practice where it is not being
+  implemented
+* make a conscious decision whether the risks we expose ourselves to are
+  acceptable
+* where we are exposed to unacceptable risk, mitigate this by ending the
+  collaboration if necessary
+
+## Vetting Process The below diagram illustrates the process that the rest of
+this document describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+We define suppliers as any organisation that we collaborate with in providing
+our clients with our services.
+
+Our Supplier List tracks the suppliers we interact with. This is checked for
+completeness monthly by the [Information Security
+Officer](../README.md#contacts). As part of this check, we make sure all
+entities that have ever invoiced us are on the Supplier List.
+
+### Supplier Ownership
+
+The [Information Security Officer](../README.md#contacts) will assign a Supplier
+Owner to each Supplier in the Supplier List. These owners are tracked in the
+Supplier List. They are responsible for conducting the Vetting Process laid out
+below.
+
+### Vetting Process
+
+#### Vetting Requirement
+
+We organise suppliers into four categories:
+
+* Organisations without access to sensitive information.
+  * These do not require a security vetting.
+* Individuals and small organisations
+  * These are likely to require an NDA to be signed and/or a negotiation around
+    how to address vulnerabilities
+* Medium-sized organisation
+  * These are likely to require a review of their publicly available info sec
+    policies, a questionnaire where necessary and negotiations to address
+    vulnerabilities where necessary
+* Large organisation
+  * These are likely to require a review of their publicly available info sec
+    policies. Given they are significantly larger than us, it is usually not
+    feasible to require a questionnaire and/or negotiations. Therefore a
+    decision to collaborate or not will be made based solely on the publicly
+    available information.
```

Comment:
There is a full-stop at the end of this sentence, whereas there is not at the end of all others.